### DISCUSSION TITLE
collection_substance.tsv and collection_compound.tsv

### DISCUSSION THREAD

**Discussion by jeevangelista** [nih-cfde/published-documentation#336] [Q&A] [answered]
Hi, are these two tables required to be filled or can it just be collection_compound? Validating with just the latter did not give me any error but the preparation script implies that the former is needed for the compound.tsv table.

**Comment by abradyIGS** [comment, marked as answer]
Only populating collection_compound is fine. When the prep script is updated to handle the newest features (end of month), it'll scan compound_substance; if it's not empty, the script will auto-add any compounds that are canonical compound IDs for the  substance IDs given in compound_substance. Either way, it's permissible to just directly list compounds in collection_compound without reference to any given substances.
